local_pathfinding/node_mock_global_path.py

```diff
@@ -0,0 +1,222 @@
+"""The mock global path node. This node is responsible for sending the mock global path to
+ Local Pathfinding"""
+
+import csv
+import os
+import time
+from typing import List
+
+import numpy as np
+import rclpy
+from custom_interfaces.msg import GPS, HelperLatLon, Path
+from custom_interfaces.srv import GlobalPath
+from rclpy.node import Node
+
+# Destination is hardcoded temporarily as a single element list
+MOCK_DESTINATION = []
+# List of doubles is an allowed type for a ros2 parameter
+MOCK_DESTINATION.append("49.263,123.138")
+# Mock gps data to get things running until we have a running gps node
+MOCK_GPS = GPS(lat_lon=HelperLatLon(latitude=48.9594, longitude=-123.3634))
+
+NUM_INTERVALS = 100
+GLOBAL_PATH_UPDATE_DELAY = 10
+
+
+def main(args=None):
+    rclpy.init(args=args)
+    mock_global_path = MockGlobalPath()
+
+    rclpy.spin(node=mock_global_path)
+
+    mock_global_path.destroy_node()
+    rclpy.shutdown()
+
+
+class MockGlobalPath(Node):
+    """Stores and sends the global path to the navigate node.
+
+    Subscribers:
```

Comment:
Maybe we can be more specific in `node_*.py`: topic subscribers/publishers, service clients/servers